### PR TITLE
Update travis to Xcode 9.0

### PR DESCRIPTION
**Pull request by AtomicCat**

### CODE REVIEW

**Review by ejensen — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -9,17 +9,17 @@ env:
...
+  - IOS_SDK=iphonesimulator11.0
+  - OSX_SDK=macosx10.13
+  - TVOS_SDK=appletvsimulator11.0
+  - WATCHOS_SDK=watchsimulator4.0
   - EXAMPLE_SCHEME="iOS Example"
   matrix:
-    - DESTINATION="OS=10.2,name=iPhone 6S"                  SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="YES"
-    - DESTINATION="OS=10.2,name=iPhone 5"                   SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO"
+    - DESTINATION="OS=11.0,name=iPhone 7"                   SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="YES"
+    - DESTINATION="OS=11.0,name=iPhone 5S"                  SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO"
```
Comment (cnoon) [inline .travis.yml:19]: Hmmmm...we should either run tests against the iOS `10.2` or `10.3` simulator instead for the 5S. Then we're getting more coverage for the other platforms.

File: CHANGELOG.md [hunk 2]
```diff
@@ -61,10 +61,16 @@ All notable changes to this project will be documented in this file.
...
 - Copyright dates to 2017.
   - Added by [Dave Camp](https://github.com/atomiccat) in Pull Request [#27](https://github.com/Nike-Inc/Willow/pull/27).
+- ExecutionMethod enum casing in example.
+  - Added by [Eric Jensen](https://github.com/ejensen) in Pull Request [#22](https://github.com/Nike-Inc/Willow/pull/22).
+
 
 #### Fixed
 
 - Compile error for Swift 3.2/4.0.
+  - Added by [Dave Camp](https://github.com/atomiccat) in Pull Request [#27](https://github.com/Nike-Inc/Willow/pull/27).
```
Comment (cnoon) [inline CHANGELOG.md:71]: `Added by` -> `Fixed by`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -61,10 +61,16 @@ All notable changes to this project will be documented in this file.
...
+- Remove redundant cast from Date to Date
+  - Added by [Eric Jensen](https://github.com/ejensen) in Pull Request [#23](https://github.com/Nike-Inc/Willow/pull/23).
```
Comment (cnoon) [inline CHANGELOG.md:73]: `Added by` -> `Fixed by`?

File: CHANGELOG.md [hunk 4]
```diff
@@ -61,10 +61,16 @@ All notable changes to this project will be documented in this file.
...
   - Added by [Dave Camp](https://github.com/atomiccat) in Pull Request [#27](https://github.com/Nike-Inc/Willow/pull/27).
+- ExecutionMethod enum casing in example.
+  - Added by [Eric Jensen](https://github.com/ejensen) in Pull Request [#22](https://github.com/Nike-Inc/Willow/pull/22).
+
 
 #### Fixed
 
 - Compile error for Swift 3.2/4.0.
+  - Added by [Dave Camp](https://github.com/atomiccat) in Pull Request [#27](https://github.com/Nike-Inc/Willow/pull/27).
+- Remove redundant cast from Date to Date
```
Comment (cnoon) [inline CHANGELOG.md:72]: `Removed redundant cast` and needs `.` at the end.

**Review by cnoon — CHANGES_REQUESTED** [review]
A few small comments back to you @AtomicCat.

**Comment by cnoon** [comment]
I made the changes called out in my review and pushed them into `master` in commits 47a03d6e and 7d5a547f which keeping your attribution @AtomicCat. 👍🏻

**Pull request closed without merging**